schema.sql

```diff
@@ -15,3 +15,54 @@ create table books (
   author_id integer references authors (id)  
 );
 
+-- a blog with users and posts
+-- a user has posts
+
+create table author (
+  id serial primary key,
+  name text
+);
+
+create table posts (
+  id serial primary key,
+  title text,
+  word_count integer,
+  category text,
+  publish_date timestamp,
+  author_id integer references author (id)
+);
+
+-- QUIZ APP ============================
+-- A quiz app, with questions and answers
+--a questions has an answer
+
+create table answer (
+  id serial primary key,
+  solution text
+);
+
+create table question (
+  id serial primary key,
+  quiz_question text,
+  difficulty integer,
+  category text, 
+  answer_id integer references answer(id)
+);
+
+-- PING PONG ============================
+-- A ping-pong tournament app, with users and games
+
+-- a game has users
+create table players (
+  id serial primary key,
+  name text,
+  skill_level integer
+);
+
+create table game (
```

Comment:
Likewise, maybe pluralize `games`? Also, consider capturing the scores of the different players.